Resources/Prototypes/Entities/Mobs/Player/familiars.yml

```
@@ -81,6 +81,8 @@
     name: ghost-role-information-remilia-name
     description: ghost-role-information-remilia-description
     rules: ghost-role-information-remilia-rules
+    mindRoles:
+    - MindRoleGhostRoleFamiliar
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)